src/scratch/asteroids/README.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Lærerveiledning - Asteroids
+level: 4
+language: nb
+---
+
+# Om oppgaven {.activity}
+På slutten av 1970-tallet ga Atari ut to spill hvor man skulle
+kontrollere et romskip. Det første var Lunar Lander, men dette ble
+utkonkurrert av Asteroids som Atari ga ut noen måneder senere.
+Spillene var faktisk så like at de kunne gjenbruke mye av teknologien.
+Vi skal gjøre det samme! Du må derfor ha laget
+[Lunar Lander](../lunar_lander/lunar_lander.html) før du begynner på
+dette prosjektet. I Asteroids er målet å beskytte romskipet mot
+asteroider ved å skyte dem i småbiter.
+
+![](asteroids.png)
+
+## Oppgaven passer til: {.check}
+ __Fag__: Kunst og håndtverk, matematikk, programmering.
+__Anbefalte trinn__: 3.-10. trinn.
+__Tema__: Koordinatsystem, bildebehandlingsprogram, videreutvikling av programkode.
+__Tidsbruk__: Dobbelttime eller mer.
+
+## Kompetansemål {.challenge}
+- [ ] __Kunst og håndtverk, 4. trinn__: bruke enkle funksjoner i digitale bildebehandlingsprogram
+- [ ] __Matematikk, 4. trinn__: lese av, plassere og beskrive posisjoner i rutenett, på kart og i koordinatsystemer, både med og uten digitale verktøy
+- [ ] __Matematikk, 7. trinn__: beskrive plassering og flytting i rutenett, på kart og i koordinatsystem, med og uten digitale hjelpemidler, og bruke koordinater til å beregne avstander parallelt med aksene i et koordinatsystem
+- [ ] __Programmering, 10. trinn__: bruke grunnleggende prinsipper i programmering, slik som løkker, tester, variabler, funksjoner og enkel brukerinteraksjon
+- [ ] __Programmering, 10. trinn__: overføre løsninger til nye problemer ved å generalisere og tilpasse eksisterende programkode og algoritmer.
+
+## Forslag til læringsmål {.challenge}
+- [ ] Elevene kan tegne ulike landskap som grunnlag for et spill.
+- [ ] Elevene kan sette sammen ulike former i digitalt bildebehandlingsprogram.
+- [ ] Elevene kan plassere elementer i bestemte posisjoner i et koordinatsystem.
+- [ ] Elevene kan bruke kode til å beskrive flytting av elementer i et koordinatsystem.
+- [ ] Elevene kan bruke kode til å få figurer til å interagere med hverandre.
+- [ ] Elevene kan lage et spill basert på kontinuerlig brukerinteraksjon.
+- [ ] Elevene kan videreutvikle eksisterende programkode til et nytt spill.
+
+## Forslag til vurderingskriterier {.challenge}
+- [ ] Eleven viser middels måloppnåelse ved å fullføre oppgaven.
+- [ ] Eleven viser høy måloppnåelse ved å videreutvikle egen kode basert på oppgaven.
```

Comment:
"... for eksempel ved å gjøre en eller flere av variasjonene nevnt nedenfor." eller vil vi at dette forslaget til vurdering skal være standardisert slik som her?